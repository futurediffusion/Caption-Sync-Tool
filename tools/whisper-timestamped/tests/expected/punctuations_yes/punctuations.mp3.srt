1
00:00:00,380 --> 00:00:02,760
Dis-moi, est-ce que l'avion vole ?

