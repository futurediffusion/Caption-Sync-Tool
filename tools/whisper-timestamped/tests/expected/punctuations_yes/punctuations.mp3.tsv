start	end	text
380	2760	Dis-moi, est-ce que l'avion vole ?
